backend/controllers/user.js

```diff
@@ -5,17 +5,17 @@ import Team from "./../models/teamModel.js";
 import mongoose from "mongoose";
 
 // @desc    Get user profile
-// @route   Get /api/users/profile
-// @access  Private
+// @route   Get /api/users/profile?_id=123456
+// @access  Public
 const getUserProfile = expressAsyncHandler(async (req, res) => {
```

Comment:
you can remove this function completely as we already have `getUserProfileByUsername` function for this purpose